code/headers/qik_2s12v10.hpp

```diff
@@ -6,7 +6,7 @@
 
 /// @file
 
-namespace r2d2::moving_platform {
+namespace r2d2 {
```

Comment:
it still needs to be r2d2::moving_platform

Reply:
this is added in the last commit